src/components/countryDetails/CountryDetails.tsx

```diff
@@ -1,45 +1,214 @@
-import {
-	Link,
-	LoaderFunction,
-	useLoaderData,
-	useParams,
-} from "react-router-dom";
+import { useEffect } from "react";
+import { LoaderFunction, useLoaderData } from "react-router-dom";
 import { useCountriesContext } from "../../Context/CountriesDataContext";
+import {
+	StyledArrowIcon,
+	StyledCountryName,
+	StyledDivWithButtons,
+	StyledDivWithCountryData,
+	StyledDivWithCountryInfo,
+	StyledDivWithFlag,
+	StyledFlagImage,
+	StyledLeftColumnWithCountryInfo,
+	StyledLink,
+	StyledLinkWithBorderCountries,
+	StyledParagraphWithCountryInfo,
+	StyledSection,
+	StyledSpanWithCategoryName,
+} from "./CountryDetails.styles";
+import { CountriesProps } from "../CountriesList/CountriesProps";
 
 export interface CountryDataTypes {
```

Comment:
This component is wwwwaaaaayyyyyy too long. At the begining try to move the type to external file

Reply:
fixed.